### ISSUE TITLE
hostpath driver must implement GetPluginCapabilities

### ISSUE THREAD

**Issue by pohly** [kubernetes-csi/drivers#68]
The example from https://github.com/kubernetes-csi/docs/pull/11 fails with the current quay.io/k8scsi images. The external-attacher runs into a fatal error and terminates:
```
I0314 16:24:49.735790       1 controller.go:75] GRPC call: /csi.v0.Identity/GetPluginCapabilities
I0314 16:24:49.735800       1 controller.go:76] GRPC request: 
I0314 16:24:49.736568       1 controller.go:78] GRPC response: capabilities:<service:<> > 
I0314 16:24:49.736620       1 controller.go:79] GRPC error: <nil>
F0314 16:24:49.736636       1 controller.go:198] no plugin controller service support detected
```

This is querying the hostpath-driver, which says:
```
I0314 16:26:17.887709       1 utils.go:96] GRPC call: /csi.v0.Identity/GetPluginCapabilities
I0314 16:26:17.887761       1 utils.go:97] GRPC request: 
I0314 16:26:17.887778       1 identityserver-default.go:53] Using default capabilities
I0314 16:26:17.887804       1 utils.go:102] GRPC response: capabilities:<service:<> > 
```

I haven't looked at the source, but it looks like the hostpath driver doesn't implement GetPluginCapabilities?

**Comment by pohly** [comment]
I can reproduce it with latest kubernetes-csi/drivers and will have a look at which commit broke this.

**Comment by pohly** [comment]
It has always has been this way? Perhaps the external-attacher just didn't check?

**Comment by sbezverk** [comment]
@pohly should be fixed when #69 gets merged. hostpath plugin used a default Identity Server which was advertising Service_UNKNOWN plugin capability. It used to work because neither provisioner nor attacher checked it. Since this behaviour was changed in provisioner and attacher, provisioner stopped calling CreateVolume since it is Controller serving call.
